### DISCUSSION TITLE
Enabling Default to PR title for squash merge commits prevents disabling of Allow squash merging

### DISCUSSION THREAD

**Discussion by asyncon** [community/community#19795] [Pull Requests]
There is a **bug** in the UI where if you enable `Default to PR title for squash merge commits` then attempt to disable `Allow squash merging`, the UI will update and `Allow squash merging` appears enabled and `Default to PR title for squash merge commits` will disappear. Refreshing the page will show both `Allow squash merging` and `Default to PR title for squash merge commits` are enabled.

The work around is simply to disable `Default to PR title for squash merge commits` before disabling `Allow squash merging`. 

Disabling a feature should supersede it's sub-features, regardless of the current value.

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
